vignettes/TADAGeospatialFunctions.Rmd

```diff
@@ -0,0 +1,236 @@
+
+```{r install_TADA, eval = F, results = 'hide', message = FALSE, warning = FALSE}
+remotes::install_github("USEPA/TADA",
+  ref = "R5_package_demo",
+  dependencies = TRUE
+)
+```
+
+Finally, use the **library()** function to load the TADA R Package into your R session.
+
+```{r library, results = 'hide', message = FALSE, warning = FALSE}
+library(TADA)
+```
+
+## Help pages
+
+All TADA R package functions have their own individual help pages, listed on the [Function reference](https://usepa.github.io/TADA/reference/index.html) page on the GitHub site. Users can also access the help page for a given function in R or RStudio using the following format (example below): `?TADA::[name of TADA function]`
+
+```{r help_pages, message = FALSE}
+# Access help page for TADA_DataRetrieval
+?TADA_DataRetrieval
+```
+
+## Geospatial Functions in TADA
+
+This vignette represents three functions aimed towards completing the TADA
+Module 2 goal: *Match WQP sites with ATTAINS assessment units. Assist user
+with reviewing sites and Assessment Units (AUs) on a map and deciding to keep
+or exclude sites (systematically first, maybe using catchments as a buffer
+around the AU points/line/polygons for consistency, and then let users make
+manual edits to include or exclude sites (main requirement)). Consider adding
+option to create new AUs for new areas where WQP sites are available but no AU
+exists yet (secondary requirement). Output is a simple spreadsheet linking
+ATTAINS AU IDs with WQP Location IDs.*
+
+**A Note About ATTAINS:**
+
+The Assessment, Total Maximum Daily Load (TMDL) Tracking and Implementation
+System ([ATTAINS](https://www.epa.gov/waterdata/attains)) is an online
+platform that organizes and combines each state’s Clean Water Act reporting
+data into a single data repository. The geospatial component of ATTAINS
+includes spatial representations of each state’s assessment units as well as
+their assigned designated uses, their most recent EPA reporting category
+(i.e., their impairment status), their impaired designated uses, and the
+parameter(s) causing the impairment.
+
+Within an assessment unit, the water quality standards remain the same and all
+water features are assessed as one entity. Depending on the state, these
+assessment units can be a specific point along a river, a river reach, an
+entire river, or even an entire watershed. (In other words, assessment units
+can take the form of point, line, and area features, or some combination of
+all of them.) Moreover, it is possible that some assessment units are not
+geospatially referenced at all, meaning they are not captured in the ATTAINS
+geospatial database.
+
+## `TADA_MakeSpatial()`
+
+This function converts any Water Quality Portal (WQP)-style data frame with
+latitude/longitude data into an {sf} object and is the backbone for future
```

Comment:
```suggestion
latitude/longitude data into an sf object and is the backbone for future
```

The TADA documentation doesn't seem to have the same package notation as we use internally so I'd suggest matching what's done in other docs they've produced: removing curly braces around package names from this doc and our function definitions.

Reply:
will modify to conform to TADA-style